java/code/src/com/redhat/rhn/frontend/dto/PackageOverview.java

```diff
@@ -37,6 +37,7 @@
     private String version;
     private String epoch;
     private String release;
+    private boolean retracted;
```

Comment:
I think this might be also one of those cases where we need to use `Boolean` since those overview classes are generally used by many queries which don't always fill the whole object.

Reply:
You're right (e.g. `packages_available_to_errata_in_channel` does not return the `retracted`). Let me change it.